packages/rich-text/src/hooks/useCustomEditor.ts

```diff
@@ -0,0 +1,8 @@
+import { useSlateStatic } from 'slate-react';
+import { CustomEditor } from '../types';
+
+export function useCustomEditor() {
+  const editor = useSlateStatic() as CustomEditor;
+
+  return editor;
+}
```

Comment:
what does this do?

Reply:
Did this to have the editor as a CustomEditor type, to facilitate this type conversion.